include/file.iol

```diff
@@ -77,6 +77,20 @@ type ListResponse:void {
 interface FileInterface {
 RequestResponse:
 	/**!
+	 * Constructs an absolute path to the target file or directory. 
+	 * Can be used to construct an absolute path for new files that does not exist yet.
+	 * Throws a fault if input is a relative path is not system recognized path.
+	 */
+	toAbsolutePath( string )( string ),
```

Comment:
Should declare that it can throw InvalidPathException, with its type (probably JavaExceptionType?). Same for the one below.